packages/node/src/lib/process-response.ts

```diff
@@ -62,12 +58,12 @@ export default function processResponse(
     //
     // This is the same thing that Node.js does internally:
     // https://github.com/nodejs/node/blob/9b8ba2536044ae08a1cd747a3aa52df7d1815e7e/lib/_http_server.js#L318
-    statusText: res.statusMessage || STATUS_CODES[res.statusCode],
+    statusText: (res.statusMessage || STATUS_CODES[res.statusCode]) as string,
```

Comment:
why not `res.statusMessage || STATUS_CODES[res.statusCode] || ''`?

Basically I feel like every `as` is a possible error and should be avoided if possible; in this case the contract is that the function returns a string for `statusText`, so shouldn't we be able to convert it safely? (Or alter the contract if not!)

Reply:
Agreed! I don't love the `as` usage but there's a broader question of what kind of data our Metrics ingestion service considers valid. I err'd on the side of `as` over fallback data that the service might not like, hence I'd like a review from @gratcliff 😇

Reply:
but per your point I am doing a second pass and removing the `as` usage where I can, see https://github.com/readmeio/metrics-sdks/pull/900/commits/8e1b743943a1ebfc03cbdf978d72cebacc8fa226